plugins/ReverbSC/revsc.c

```diff
@@ -65,8 +65,8 @@ int sp_revsc_init(sp_data *sp, sp_revsc *p)
     p->dampFact = 1.0;
     p->prv_LPFreq = 0.0;
     p->initDone = 1;
-    int i, nBytes = 0;
-    for(i = 0; i < 8; i++){
+	int i = 0, nBytes = 0;
+	for(i = 0; i < 8; i++){
```

Comment:
These got converted to tabs where the rest of this file seems to be spaces. Shall we convert the rest of the file to tabs?

Reply:
I guess we could.